src/main/java/pl/confitura/jelatyna/email/EmailService.java

```diff
@@ -0,0 +1,21 @@
+package pl.confitura.jelatyna.email;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import pl.confitura.jelatyna.admin.Admin;
+
+@Service
+public class EmailService {
+
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    @Autowired
+    private Sender sender;
+
+    public void adminCreated(Admin admin) {
+        sender.send(admin.getEmail(), "admin-creation",
```

Comment:
ale co to znaczy good practice? to pole jest użyte w jednym miejscu. jaki zysk będzie z przesunięcia stringa `"admin-creation"` to pola `ADMIN_CREATION`?
wrzucanie wartości do pól statycznych, ma moim zdaniem, sens jeśli:

1. jest to jakaś magiczna, niezrozumiała wartosć, np `12`. Wtedy warto przesunąć ją to pola z jakąś znaczącą nazwą. np `NO_OF_MONTHS`
2. jeśli wartość jest używana w więcej niż jednym miejscu - usunięcie duplikacji.

wydzielanie pól statycznych tylko po to bo to `good practice` jest dość dziwne :)

co do wrzucenia do propertiesa... można, ale tak naprawdę to po co?